parse.go

```diff
@@ -227,6 +228,30 @@ func parseListLine(line string, now time.Time, loc *time.Location) (*Entry, erro
 	return nil, errUnsupportedListLine
 }
 
+func parseQuotas(line string) (entries map[string]string, err error) {
+	entries = make(map[string]string)
+
+	// Regex Sample https://regex101.com/r/lv2nWj/1/
+	var re = regexp.MustCompile(`(?m)^(\s*)([a-zA-Z\-\_0-9\\\/\.\s\[\]\s]*):(\s*)([a-zA-Z\-\_0-9\\\/\.]*)$`)
+	var link = regexp.MustCompile(`(^[A-Za-z])|(_|\s)([A-Za-z])`)
+	for i, match := range re.FindAllStringSubmatch(line, -1) {
```

Comment:
Do you really need regular expressions to parse simple key-value lines?

Reply:
Can you recommend a different way to parse this?

Reply:
Why not `strings.Split(line, ":"`) ?